lte/cloud/go/tools/migrations/m013_policy_ipv6/types/new.go

```diff
@@ -0,0 +1,122 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package types
+
+// PolicyRuleConfig policy rule config
+// swagger:model policy_rule_config
+type PolicyRuleConfig struct {
```

Comment:
Use `json.RawMessage` as the type for fields that you don't need to inspect (so everything other than `FlowList` I think). This will also save you from defining some nested structs